apps/backend/src/modules/activity-logs/activity-log.service.ts

```diff
@@ -178,8 +177,10 @@ class ActivityLogService implements Service {
 
 				return {
 					commitsNumber: commitsArray,
-					contributorId: contributorId ?? "",
-					contributorName: contributorName ?? "",
+					contributor: {
+						id: contributorId ?? "",
+						name: contributorName ?? "",
```

Comment:
We know for sure, that their are strings
```suggestion
						id: contributorId as string,
						name: contributorName as string,
```

Reply:
Also, I thought about it, but previously, it was like this. I just wrapped it with contributor and planned to revisit it later.